plugins/source/pagerduty/docs/tables/pagerduty_addons.md

```diff
@@ -0,0 +1,23 @@
+# Table: pagerduty_addons
+
```

Comment:
(Applies to all tables, not a blocker) It would be nice to link to the API docs in the table description